### ISSUE TITLE
ImportError: cannot import name 'Literal' from 'typing'

### ISSUE THREAD

**Issue by geoffagnew** [ehossack/django-backblaze-b2#5]
Hi,

I'm trying to use django-backblaze-b2 with my app but am encountering this error when I add `django_backblaze_b2` to my `INSTALLED_APPS`. 

I'm on Python 3.7.4 and Django 3.0.5

```
File ".../lib/python3.7/site-packages/django_backblaze_b2/options.py", line 1, in <module>
    from typing import Any, Dict, Literal, Optional, TypedDict, Union
ImportError: cannot import name 'Literal' from 'typing' (/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/typing.py)
```
According to the docs `typing.Literal` is new in Python version 3.8. The docs for django-backblaze-b2 says Python 3.7 is supported? Am I missing something?

**Comment by ehossack** [comment]
Nope, you're absolutely right, oversight on my part. I upgraded after the end of official version support this june.

Are you able to upgrade to 3.8, or do you need to remain on 3.7.X?

**Comment by geoffagnew** [comment]
I probably won't upgrade for this. Whether you remove the typing is entirely your call. If it's not removed may I suggest updating the supported Python versions in the docs so other folks on 3.6 or 3.7 don't make the same error I did.

Thanks.

**Comment by ehossack** [comment]
Fixed in v1.1.3
